packages/theming/atlas/src/themesource/atlas_core/web/layouts/_layout-atlas-responsive.scss

```diff
@@ -23,7 +23,7 @@
                     &.mx-navigationtree-has-items:hover > ul {
                         position: absolute;
                         z-index: 100;
-                        top: 0;
+                        top: $topbar-minimalheight;
```

Comment:
@RonnieMendix this seems to be very break as it will start with a margin top as the same height as the topbar but related to the menu, which has its own position: relative. Can you explain me why this change is necessary?

Reply:
The menu does not have its own position relative… So it will look at the relativeness of the whole screen, and then you can add the top distance the same as the topbar.

But ow you mention it, what we also can do, is adding position: relative; to the mx-scrollcontainer-middle, because that is where the menu is located. Then top: 0; for the submenu will also work and show the submenu under the topbar. (which probably is more safe if you don't want a topbar and doesn't set the variable [topbar height] to 0)

Reply:
@diego-antonelli  Do you want me to commit a version with position relative instead of the topbar height as top spacing?